homeassistant/components/media_player/sonos.py

```diff
@@ -253,3 +258,14 @@ def turn_on(self):
     def play_media(self, media_type, media_id):
         """Send the play_media command to the media player."""
         self._player.play_uri(media_id)
+
+    def _is_reachable(self):
+        """Return True if player is reachable, False otherwise."""
+        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        try:
+            sock.connect((self._player.ip_address, 1443))
```

Comment:
even the default timeout would be too long since we only want to check if the device is there or not (ping), a number between 2 and 5sec at max should be fine